UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Models/Response/BaseCallBackResponse.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+
+namespace UKHO.ExchangeSetService.Common.Models.Response
+{
+    public class BaseCallBackResponse
+    {
+        [JsonProperty("specversion")]
+        public string SpecVersion { get; set; }
+        [JsonProperty("type")]
+        public string Type { get; set; }
+        [JsonProperty("source")]
+        public string Source { get; set; }
+        [JsonProperty("id")]
+        public string Id { get; set; }
+        [JsonProperty("time")]
+        public string Time { get; set; }
+        [JsonProperty("subject")]
+        public string Subject { get; set; }
+        [JsonProperty("datacontenttype")]
```

Comment:
As above should this be dataContentType?

Reply:
No, as per the exchange set service API spec this is correct format.
Reference ESS endpoint :-  /callback

Reply:
Thanks - Makes sense!